message_post_model/__openerp__.py

```diff
@@ -34,7 +34,8 @@
     ],
     "demo": [],
     "data": [
-        "security/ir.model.access.csv"
```

Comment:
The same answer:
https://github.com/Vauxoo/addons-vauxoo/pull/1075#discussion_r83271412